.gitlab-ci.yml

```diff
@@ -162,16 +162,35 @@ variables:
   # Build images versions
   # To use images from datadog-agent-buildimages dev branches, set the corresponding
   # SUFFIX variable to _test_only
-  DATADOG_AGENT_BUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_BUILDIMAGES: v32377621-ce42cb8e
-  DATADOG_AGENT_WINBUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_WINBUILDIMAGES: v32377621-ce42cb8e
-  DATADOG_AGENT_ARMBUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_ARMBUILDIMAGES: v32377621-ce42cb8e
-  DATADOG_AGENT_SYSPROBE_BUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_SYSPROBE_BUILDIMAGES: v32377621-ce42cb8e
-  DATADOG_AGENT_BTF_GEN_BUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_BTF_GEN_BUILDIMAGES: v32377621-ce42cb8e
+  CI_IMAGE_AGENT_DEPLOY_SUFFIX: ""
```

Comment:
Would you mind clarifying the end goal and why this is necessary to get there?
I see you've global search/replaced, but any external use of these variables will be broken by this change.

For example. local scripts which use the `....IMAGES` entry in `.gitlab-ci.yml` to determine what container to pull locally.

Reply:
Hi 👋🏽 
The goal is to be able to build only the image(s) concerned by your changes in the `datadog-agent-buildimages` repository, instead of building everything at each PR commit.
I changed the naming convention to make the variable names more concise, I'll check for parts of the naming to find if usage of the name pattern is done elsewhere.
Thanks for your remark

Reply:
@derekwbrown I only found an occurrence on `omnibus` script. Do you have other places where I should check if a pattern matching was done on these image names?
Many thanks in advance

Reply:
So, there are a couple of places
1) We have other repos that use the resulting images from `datadog-agent-buildimages`.  I have historically structured those to use the same variables, so that they could be updated easily, easy to check/compare, etc.
Such as
https://github.com/DataDog/datadog-windows-procmon/blob/main/.gitlab-ci.yml#L17-L24

2) I have (perhaps others do) local scripts which parse the file.

While I agree your naming is probably better than the original, I suspect that this will have some downstream consequences that we cannot foresee.